packages/kit-headless/package.json

```diff
@@ -10,6 +10,9 @@
   "module": "./index.qwik.mjs",
   "types": "./index.d.ts",
   "type": "module",
+  "scripts": {
+    "setup:chrome:113": "npx tsx ./scripts/prepare-chrome-113.ts"
```

Comment:
Naming feels a bit verbose but there's no doubt what's going on here.

Reply:
I think it's good but let's use the dot notation to stay consistent with the other scripts
Plus, maybe we should even move this to the root package.json, as we have all of our scripts there anyway

WDYT?

Reply:
I changed it to be dot notation. I'm a bit torn on where it should live. I was on the fence about where to put the script. The script is very scoped to those tests so I didn't want to add complexity to the upper `package.json` if it really was this isolated. I think if we add another test package that requires chrome 113, we can refactor it up. The other nice benefit is that I can use it as a NX task dependency when it's in the same package

![image](https://github.com/qwikifiers/qwik-ui/assets/942415/42ff4212-3aff-4e16-af3e-dc59aea82821)